src/store/NotificationStore.tsx

```diff
@@ -34,13 +35,29 @@ class NotificationStore {
       if (numberOfUnreadChats !== undefined) {
         this.unreadChats = numberOfUnreadChats;
       }
-    } catch(error) {
+    } catch (error) {
       console.error("Error while fetching number of unread chats");
     }
   }
 
+  setunreadChats(unseenChatsAsClient: number, unseenChatsAsCaretaker: number) {
```

Comment:
Mała poprawke w nazwie funkcji
```suggestion
  setUnreadChats(unseenChatsAsClient: number, unseenChatsAsCaretaker: number) {
```